docs/linux/zbd-api.md

```diff
@@ -539,36 +502,30 @@ if (ret)
 ...
 ```
 
-The device file descriptor `fd` must be open for writing for this command to
-succeed.
+The device file descriptor `fd` must be open for writing in order for this command to succeed.
 
 The  command line utility [`blkzone`](../projects/util-linux.md#blkzone) uses
 the *BLKRESETZONE* command to implement its *reset* functionality.
 
 ### Opening, Closing and Finishing Zones
 
-Explicitely opening a zone or a range of zones can be done using the
+Explicitly opening a zone or a range of zones can be done using the
 *BLKOPENZONE* command. This command uses the same arguments as the
-*BLKRESETZONE* command. It takes a pointer to a data structure `blk_zone_range`
-which must specifies the range of zones to operate on.
+*BLKRESETZONE* command. It takes a pointer to a data structure `blk_zone_range`,
+which specifies the range of zones to operate on.
 
-Closing a zone is done using the command *BLKCLOSEZONE*. Finishing a zone, that
-is transitioning the zone to the *full* condition (`BLK_ZONE_COND_FULL`), is
-done using the *BLKFINISHZONE* command. Both of these commands also take as an
-argument a pointer to a `blk_zone_range` data structure to specify the range of
-zones to operate on.
+Closing a zone is done using the command *BLKCLOSEZONE*. Finishing a zone--that
+is, transitioning the zone to the *full* condition (`BLK_ZONE_COND_FULL`), is
+done using the *BLKFINISHZONE* command. Both of these commands also take as 
+arguments a pointer to the `blk_zone_range` data structure to specify the range of zones to operate on.
 
 The *BLKOPENZONE*, *BLKCLOSEZONE* and *BLKFINISHZONE* commands were introduced
-with kernel version 5.5.0.
+in kernel version 5.5.0.
 
 ### Zone Size and Number of Zones
 
-Linux&reg; kernel version 4.20 introduced two new additional commands to obtain
-a zoned device zone size (`BLKGETZONESZ`) and the total number of zones of the
-device (`BLKGETNRZONES`). Both commands take a pointer to an unsigned 32-bits
-integer variable as argument where the zone size value or the number of zones
-will be returned. The following sample C code illustrates the use of these
-commands.
+Linux&reg; kernel version 4.20 introduced two more new commands: one to obtain
```

Comment:
Maybe: s/two more new commands/two new commands ?
The "more" (as was additional in the previous text) seems redundant since the commands are "new" :)